packages/rule-http-compression/src/rule.ts

```diff
@@ -259,8 +264,15 @@ export default class HttpCompressionRule implements IRule {
         };
 
         const checkBrotli = async (resource, element) => {
-            const { contentEncodingHeaderValue, rawResponse, response } = await getNetworkData(resource, { 'Accept-Encoding': 'br' });
+            let networkData = await getNetworkData(resource, { 'Accept-Encoding': 'br' });
+
+            if (!networkData) {
+                await context.report(resource, element, 'Error fetching brotli content');
```

Comment:
Maybe we should change these types of messages to something more user friendly? 

e.g.: `Couldn't fetch content when Brotli compression is requested` ?